.gitignore

```diff
@@ -13,5 +13,6 @@ npm-debug.log
 package-lock.json
 test.json
 yarn.lock
+.env
```

Comment:
Nitpick: .env -> .env.template

Reply:
@syerrapragada , since we expect everybody to have their version of `.env` - .env needs to go in `.gitignore` to not include track any changes to it. We do want to track any changes to .env.template!